tools/distrib/python/grpcio_tools/grpc/__init__.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python2.7
```

Comment:
Like this `httpie` `flask` `django` `requests` and `scrapy` not use shebang, so closed this pr. Thanks.